.github/workflows/integration-tests-smoke.yml

```diff
@@ -27,40 +27,31 @@ env:
 
 jobs:
   build_chainlink_image:
-    name: Build Chainlink Image ${{matrix.image.name}}
+    name: Build Chainlink Image
     runs-on: ubuntu-latest
     env:
       GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     environment: integration
     permissions:
       id-token: write
       contents: read
-    strategy:
-      matrix:
-        image:
-          - name: ""
-            dockerfile: core/chainlink.Dockerfile
-            tag-suffix: ""
-          - name: (plugins)
-            dockerfile: plugins/chainlink.Dockerfile
-            tag-suffix: -plugins
     steps:
       - name: Collect Metrics
         id: collect-gha-metrics
         uses: smartcontractkit/push-gha-metrics-action@d9da21a2747016b3e13de58c7d4115a3d5c97935 # v3.0.1
         with:
-          id: starknet-e2e-build${{ matrix.image.tag-suffix }}
+          id: starknet-e2e-build
           org-id: ${{ secrets.GRAFANA_INTERNAL_TENANT_ID }}
           basic-auth: ${{ secrets.GRAFANA_INTERNAL_BASIC_AUTH }}
           hostname: ${{ secrets.GRAFANA_INTERNAL_HOST }}
-          this-job-name: Build Chainlink Image${{matrix.image.name}}
+          this-job-name: Build Chainlink Image
         continue-on-error: true
```

Comment:
Any references to this can be fully removed by the way. This isn't blocking, but for future references.